app/decorators/visit_decorator.rb

```diff
@@ -7,6 +7,9 @@ class VisitDecorator < Draper::Decorator
     :prisoner_availability_unknown?,
     :slot_availability_unknown?,
     :slots_unavailable?,
+    :contact_list_unknown?,
```

Comment:
@chrishanes these are the methods you can use from the view with the `@visit` object